### PR TITLE
Adding in Mumbai/Mainnet precursor deb packaging for tests to use dur…

### PR DESCRIPTION
**Pull request by djpolygon**
…ing upgrade(iterations to come)

# Description

Please provide a detailed description of what was done in this PR

# Changes

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ d] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

This change implements packaging for .deb and introduces profile concepts for installing for auto creating sentry nodes/validators. These packages rely on new hosts to be used on peers.

# Checklist

- [ ] I have added at least 2 reviewer or the whole pos-v1 team
- [ ] I have added sufficient documentation in code
- [ ] I will be resolving comments - if any - by pushing each fix in a separate commit and linking the commit hash in the comment reply

# Cross repository changes

- [ x] This PR requires changes to heimdall
    - In case link the PR here:
    - https://github.com/maticnetwork/heimdall/pull/875
- [ ] This PR requires changes to matic-cli
    - In case link the PR here:

## Testing

- [ ] I have added unit tests
- [ ] I have added tests to CI
- [ ] I have tested this code manually on local environment
- [ ] I have tested this code manually on remote devnet using express-cli
- [x ] I have tested this code manually on mumbai
- [ ] I have created new e2e tests into express-cli

### Manual tests

Please complete this section with the steps you performed if you ran manual tests for this functionality, otherwise delete it

# Additional comments

These tooling changes are per posv1 changes for the migration to more standardized debian packages. Other tooling changes will be in subsequent changes for clean up, rpm addition, and other packaging.

### CODE REVIEW

File: .github/workflows/packager.yml [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@master
+        with:
+          go-version: 1.19
+      - name: Generate release tag
+        id: tag
+        run: echo "::set-output name=release_tag::bor_$(date +"%Y.%m.%d_%H-%M")"
```
Comment (cffls) [inline .github/workflows/packager.yml:27]: The release name should match the release tag exactly (no date/time info should be included), as what we've been doing in the past: https://github.com/maticnetwork/bor/releases/

File: .github/workflows/packager.yml [hunk 2]
```diff
@@ -0,0 +1,318 @@
+name: packager
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - '**'
```
[the remark this answers is not in the document]
Comment (djpolygon) [reply .github/workflows/packager.yml]: Removed

File: packaging/templates/systemd/bord.service [hunk 3]
```diff
@@ -0,0 +1,36 @@
+[Unit]
```
[the remark this answers is not in the document]
Comment (djpolygon) [reply packaging/templates/systemd/bord.service]: Updated and removed

File: .github/workflows/packager.yml (hunk 1 above)
Comment (djpolygon) [reply .github/workflows/packager.yml]: Updated and now using tag

File: .github/workflows/packager.yml [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+        run: cp -rp packaging/templates/package_scripts/postrm packaging/deb/bor-mainnet-archive-config_v0.3.0-beta-arm64/DEBIAN/postrm
+      - name: Copying the toml
+        run: cp -rp packaging/templates/mainnet-v1/archive/config.toml packaging/deb/bor-mainnet-archive-config_v0.3.0-beta-arm64/var/lib/bor/
+      - name: Building porfile package
+        run: dpkg-deb --build --root-owner-group packaging/deb/bor-mainnet-archive-config_v0.3.0-beta-arm64
+
+      - name: Confirming package built
+        run: ls -ltr packaging/deb/ | grep bor
+
+      - name: Release bor Packages
```
[the remark this answers is not in the document]
Comment (djpolygon) [reply .github/workflows/packager.yml]: added

**Review by cffls — APPROVED** [review]

**Review by manav2401 — APPROVED** [review]

**Pull request merged**